src/lightning_app/cli/lightning_cli.py

```diff
@@ -125,6 +131,65 @@ def run_app(
     _run_app(file, cloud, without_server, no_cache, name, blocking, open_ui, env)
 
 
+@main.command("command")
```

Comment:
I think it is worth saying this is a beta feature and might be modified or updated in the next releases. 

Perhaps we can even have it enabled only by a feature flag (env var) for such cases? something like `ENABLE_LIGHTNING_BETA_FEATURES`

Reply:
Done !